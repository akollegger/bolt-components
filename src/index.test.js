import React from "react";
import TestRenderer from "react-test-renderer";
import { Cypher, Provider } from "./index";
import { mockDriver } from "../config/test_helpers";

it("loads Cypher", () => {
  const out = TestRenderer.create(
    <Cypher query="x" driver={mockDriver()} render={() => null} />
  );
  expect(out.toJSON()).toEqual(null);
});
it("loads Provider", () => {
  const out = TestRenderer.create(
    <Provider driver={mockDriver()}>
      <div />
    </Provider>
  );
  expect(out.toJSON()).toMatchSnapshot();
});
