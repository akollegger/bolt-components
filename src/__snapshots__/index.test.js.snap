// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads Provider 1`] = `<div />`;
